### DISCUSSION TITLE
zoom in/out key commands

### DISCUSSION THREAD

**Discussion by rngdmstr** [helge17/tuxguitar#413] [General]
Feature request: Could we get some key commands for zoom in/out? I use TG primarily for the chord diagrams which are really only useful when in maximum zoom, and I have to click View (10x clicks) --> Zoom In (10x more clicks) to see that (20x total clicks). It would be nice to just hold a key down on my keyboard and have it done in a few seconds since it's the first thing I do when I open any of my projects and it feels like a chore lol.

Thanks so much for your hard work on this, I use this software a lot and it's really useful <3

**Comment by guiv42** [comment]
Hi
feature is already present: keyboard shortcuts are configurable. Just click menu Tools/Shortcuts, then type "zoom" in first line of dialog: this shows "zoom in" and "zoom out" actions. Double-click on one action to define a customized shortcut.
![20240609_095901](https://github.com/helge17/tuxguitar/assets/129443524/3821fa43-2c8e-4c9b-93fa-9f820e63eafc)

Notes:
- please use github [issues](https://github.com/helge17/tuxguitar/issues) to create feature requests instead of discussions, as documented in [contribute doc](https://github.com/helge17/tuxguitar/blob/master/doc/contribute.md) (see "Suggest improvements" section)
- you can also zoom in/out with [Ctrl+mouse wheel] or [Ctrl+2 fingers touchpad]
